src/app/_components/GovernanceCalendarCards.tsx

```diff
@@ -105,13 +106,13 @@ export function GovernanceCalendarCards() {
             <Calendar startDate={start.dateTime} />
             <div className="flex-1 pb-14 sm:pb-0">
               <div className="flex flex-col gap-3 p-4">
-                <div className="flex gap-2">
+                <TagGroupContainer>
                   <TagLabel
-                    borderColor="brand-100"
                     icon={Clock}
                   >{`UTC ${startTime} - ${endTime}`}</TagLabel>
-                  <TagLabel>Zoom</TagLabel>
-                </div>
+                  <TagLabel variant="secondary">Zoom</TagLabel>
```

Comment:
This one could be a `TagGroup` right?

Reply:
I think because of the `icon` that we are passing, it can not. Unless we also pass through icons.

Reply:
Done. ✅